dirs/dirs.go

```diff
@@ -144,8 +144,8 @@ const (
 	// Directory with snap data inside user's home
 	UserHomeSnapDir = "snap"
 
-	// HiddenSnapDir is an experimental hidden directory for snap data
-	HiddenSnapDir = ".snap/data"
+	// HiddenSnapDataDir is an experimental hidden directory for snap data
+	HiddenSnapDataDir = ".snap/data"
```

Comment:
mmh, I think this name for clarity needs s/DataDir/DataHomeDir/ ?  that matches also the name above HiddenSnapDataHomeGlob